datafusion/physical-expr/src/aggregate/build_in.rs

```diff
@@ -939,32 +847,6 @@ mod tests {
         assert!(observed.is_err());
     }
 
-    #[test]
-    fn test_stddev_return_type() -> Result<()> {
```

Comment:
I think those tests are covered by sqllogictests testing. I didn't add similar tests for UDAF.